tests/integration/src/test/scala/mesosphere/marathon/integration/UpgradeIntegrationTest.scala

```diff
@@ -71,13 +70,18 @@ class UpgradeIntegrationTest extends AkkaIntegrationTest with MesosClusterTest w
     }
   }
 
-  case class Marathon149(suiteName: String, masterUrl: String, zkUrl: String)(
+  class Marathon149Artifact()(
```

Comment:
Hm, what do you think about making the marathon version fixtures? We cache the download so it is safe to re-download them each time.

Also, the artifact classes are so similar. I think we could just have one.